tests/conftest.py

```diff
@@ -379,6 +380,57 @@ def oidc_service(db_session):
     )
 
 
+@pytest.fixture
+def dummy_oidc_jwt():
```

Comment:
NB: Added this to de-dupe a bunch of the hard-coded test JWTs in individual tests.